docs/schemes/refresh.md

```diff
@@ -67,9 +67,13 @@ Or enable [autoRefresh](#autorefresh) option to automatically refresh tokens.
 auth: {
     strategies: {
       local: {
+        //assigning `refresh` will override `local` scheme
```

Comment:
I think we should keep this example clean without comments as all options are described below. What do you think?

Reply:
ok i'll remove the comments but I'll have to improve the documentation

Reply:
Ok, thanks! :)